app/create_app.py

```diff
@@ -71,31 +76,43 @@ def inject_global_constants():
             service_meta_author="DLUHC",
         )
 
```

Comment:
Where is talisman initiated?

Reply:
in the file

Reply:
"Take my talisman out of your - --- mouth" 😡